doc/rados/operations/placement-groups.rst

```diff
@@ -435,11 +435,9 @@ If you have more than 50 OSDs, we recommend approximately 50-100
 placement groups per OSD to balance out resource usage, data
 durability and distribution. If you have less than 50 OSDs, choosing
 among the `preselection`_ above is best. For a single pool of objects,
-you can use the following formula to get a baseline::
+you can use the following formula to get a baseline
 
-                (OSDs * 100)
-   Total PGs =  ------------
-                 pool size
+  Total PGs = :math:`\frac{OSDs \times 100}{pool \: size}`
```

Comment:
@tchaikov when you shared it this morning, the rendering for this was messed up for me. When I looked again now, I see the eventual rendering fine after a blip

Reply:
probably it takes time to load MathJax and render MathML. we could use server side rendering if this is a concern. but this is a start i think.

Reply:
just saw the discussion on the \#sepia IRC channel, probably we need to add more items to our Content Security Policy directive in Nginx's settings.